apps/core/views.py

```diff
@@ -66,6 +66,13 @@ def get(self, request, *args, **kwargs):
                 ' GROUP BY core_omicsarea.name'
             ))
 
+            pixelers_infos = Pixel.objects.raw((
+                "SELECT first_name, last_name, email,"
+                " to_char(date_joined, 'YYYY-MM-DD') AS date, 1 AS id"
+                " FROM core_pixeler"
+                " ORDER BY last_name;"
```

Comment:
Here we should use the Django model, not a raw SQL query. https://docs.djangoproject.com/en/1.11/topics/db/queries/#retrieving-objects